### PR TITLE
fix(siren backends): Adjust siren backens for sct cloud-provider option

### PR DESCRIPTION
**Pull request by yarongilor**
'aws-siren' and 'gce-siren' values were added as new backends.
	Thus it should be adjusted when cloud-provider value is requested.

siren-tests corresponding PR is: https://github.com/scylladb/siren-tests/pull/713
Trello: https://trello.com/c/MAjHqcc3

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Comment by yarongilor** [comment]
Test passed ok in: https://jenkins.scylladb.com/view/siren/job/siren-tests/job/longevity-tests/job/cloud-longevity-100gb-3d/54/

**Review by Orenef11 — APPROVED** [review]
LGTM

File: vars/getCloudProviderFromBackend.groovy [hunk 1]
```diff
@@ -6,6 +6,8 @@ def call(String backend) {
         'k8s-local-kind-aws': 'aws',
         'k8s-local-kind-gce': 'gce',
         'k8s-gce-minikube': 'gce',
+        'siren-aws': 'aws',
+        'siren-gce': 'gce',
```
Comment (yarongilor) [inline vars/getCloudProviderFromBackend.groovy:10]: @vponomaryov can you please advice? perhaps i'm missing something here, i just added " 'siren-gce': 'gce' " but it didn't work so i had to use the above " cloud_provider = cloud_provider.replace('-siren', '') "

Comment (vponomaryov) [reply vars/getCloudProviderFromBackend.groovy:10]: You have typos here. Words in backend names are placed incorrectly. It must be "aws-siren" and "gce-siren".

File: vars/createSctRunner.groovy [hunk 2]
```diff
@@ -2,7 +2,6 @@
 
 def call(Map params, Integer test_duration, String region) {
     def cloud_provider = getCloudProviderFromBackend(params.backend)
-
```
Comment (vponomaryov) [inline vars/createSctRunner.groovy:5]: Change to this file looks unrelated.

File: vars/runCleanupResource.groovy [hunk 3]
```diff
@@ -32,7 +32,7 @@ def call(Map params, String region){
     fi
 
     echo "Starting to clean resources ..."
-    if [[ "$cloud_provider" == "aws" || "$cloud_provider" == "gce" || "$cloud_provider" == "k8s-local-kind-aws" || "$cloud_provider" == "k8s-local-kind-gce" ]]; then
+    if [[ "$cloud_provider" == "aws" || "$cloud_provider" == "gce" || "$cloud_provider" == "k8s-local-kind-aws" || "$cloud_provider" == "k8s-local-kind-gce" || "$cloud_provider" == "aws-siren" || "$cloud_provider" == "gce-siren" ]]; then
```
Comment (vponomaryov) [inline vars/runCleanupResource.groovy:35]: All such changes are not needed. Having "vars/getCloudProviderFromBackend.groovy" be updated is enough.
"cloud_provider" will give you "aws" or "gce". And these values are already part of the condition.
Existing check for 2 "k8s" values are redundant.

**Review by vponomaryov — CHANGES_REQUESTED** [review]

File: vars/createSctRunner.groovy (hunk 2 above)
Comment (yarongilor) [reply vars/createSctRunner.groovy:5]: removed.

File: vars/runCleanupResource.groovy (hunk 3 above)
Comment (yarongilor) [reply vars/runCleanupResource.groovy]: fixed.

**Review by vponomaryov — APPROVED** [review]

**Review by enaydanov — APPROVED** [review]
